kubectl-rook-ceph.sh

```diff
@@ -273,6 +276,94 @@ function run_purge_osd() {
     rook ceph osd remove --osd-ids=$1 --force-osd-removal=$force_removal"
 }
 
+####################################################################################################
+# 'kubectl rook-ceph debug commands
+####################################################################################################
+
+function update_deployment_spec_command(){
+  deployment_spec=$1
+  main_command='["sleep","infinity"]'
+  deployment_spec=$(jq .template.spec.containers[0].command="$main_command" <<< $deployment_spec)
+  deployment_spec=$(jq .template.spec.containers[0].args=[] <<< $deployment_spec)
+  echo "$deployment_spec" # for returning the updated deployment_spec
+}  
+
+function update_deployment_spec_image(){
+  deployment_spec=$1
+  alternalte_image=\"$2\"
+  deployment_spec=$(jq .template.spec.containers[0].image="$alternalte_image" <<< $deployment_spec)
+  deployment_spec=$(jq .template.spec.containers[0].env[0].value="$alternalte_image" <<< $deployment_spec)
+  echo "$deployment_spec" # for returning the updated deployment_spec
+}
+
+function verify_deployment_exist(){
+  if [[ $deployment_name != *"mon"* ]] && [[ $deployment_name != *"osd"* ]]; then
+    fail_error "only mon or osd deployment name can passed for starting debug mode"
+  fi
+  if output="$(KUBECTL_NS_CLUSTER get deployment "$deployment_name")"; then
+    echo "deployment $deployment_name exist, $output" 
+  else  
+    fail_error "deployment $deployment_name doesn't exist"
+  fi
+}
+
+function run_start_debug(){
+  [[ -z "${1:-""}" ]] && fail_error "Missing mon or osd deployment name"
+  deployment_name="$1"
+  alternate_image_flag="${2:-""}"
+  alternate_image="${3:-""}"
+  verify_deployment_exist "$deployment_name"
+  echo "setting debug mode for \"$deployment_name\""
```

Comment:
```suggestion
  echo "setting debug mode for \"$deployment_name\""
```
merge this echo with L326 and L335 to look something like
```
Setting debug mode for deployment <name> for container <name>
```

Reply:
There is in between image echo command too 
`echo "setting debug image to $alternate_image"`

So anyhow we need to keep it separately, right?

I did a lot's of echo so it can be easy to reproduce from the script which subtask it got failed